### PR TITLE
Add clever f

### PR DESCRIPTION
**Pull request by wsdjeg**
ref: #1436

### CODE REVIEW

File: autoload/SpaceVim.vim [hunk 1]
```diff
@@ -69,7 +69,7 @@ let g:spacevim_windows_leader          = 's'
 ""
 " Unite work flow leader of SpaceVim. Default is `f`.
```
Comment (jetm) [inline autoload/SpaceVim.vim:70]: Here should says: Default is `\f`.

File: autoload/SpaceVim.vim [hunk 2]
```diff
@@ -69,7 +69,7 @@ let g:spacevim_windows_leader          = 's'
...
 " Set to empty to disable this feature, or you can set to another char.
-let g:spacevim_unite_leader            = 'f'
+let g:spacevim_unite_leader            = '\f'
 ""
 " Denite work flow leader of SpaceVim. Default is `F`.
```
Comment (jetm) [inline autoload/SpaceVim.vim:74]: I was expecting F will be \F as well as for unite.

**Review by jetm — CHANGES_REQUESTED** [review]
I will make these changes before merge it

**Comment by wsdjeg** [comment]
maybe we can remove this option, and denite layer is same as unite layer. also same as ctrp layer, user should use only one of them, and these there layer maybe should has same key bindings. the prefix can be setted when loadding layer.

**Comment by zhigang1992** [comment]
hey guys, I'm having trouble use denite's lead key, which was `F`. Should that be mentioned somewhere and how should I fix that?

**Comment by wsdjeg** [comment]
that was removed. you can load denite layer. and read denite layer documentation.

**Pull request merged**